### ISSUE TITLE
Have advanced challenge for c/c++

### ISSUE THREAD

**Issue by commjoen** [OWASP/wrongsecrets#617]
Based on https://github.com/OWASP/wrongsecrets-binaries/issues/11

**Comment by commjoen** [comment]
@roddas will pick this up (somehow cannot assign yet :( )

**Comment by roddas** [comment]
In this issue, I should only upload the binaries, right?

Can you give  me some example of documentation please?

**Comment by commjoen** [comment]
Yes, you can upload the binaries and create a challenge around them :-)
So steps would be:
1. follow https://github.com/OWASP/wrongsecrets/blob/master/CONTRIBUTING.md#how-to-add-a-challenge to setup the new challenge (including unit tests in the end)
2. copy the new executables to src/main/resources/executables (see https://github.com/OWASP/wrongsecrets/tree/master/src/main/resources/executables) make sure they have a name that is consistent, like wrongsecrets-advanced-c, wrongsecrets-advanced-c-arm , wrongsecrets-advanced-c-linux , wrongsecrets-advanced-c-linux-arm , wrongsecrets-advanced-c-musl , wrongsecrets-advanced-c-musl-arm ,wrongsecrets-advanced-c , wrongsecrets-advanced-c-windows
3. make sure your challenge uses the BinaryExecutionHelper, like in https://github.com/OWASP/wrongsecrets/blob/master/src/main/java/org/owasp/wrongsecrets/challenges/docker/Challenge19.java#L28 and https://github.com/OWASP/wrongsecrets/blob/master/src/main/java/org/owasp/wrongsecrets/challenges/docker/Challenge19.java#L45 and https://github.com/OWASP/wrongsecrets/blob/master/src/main/java/org/owasp/wrongsecrets/challenges/docker/Challenge19.java#L39 as you can see: you don't need to capture an answer in Java source code, instead, the java app will call your executable 😉

**Comment by roddas** [comment]
Thanks .
